### DISCUSSION TITLE
Save Game handling

### DISCUSSION THREAD

**Discussion by steilerDev** [webrcade/webrcade#14] [Q&A] [answered]
Hi All, first and foremost: Awesome project!!

One thing I did not quite understand yet (and I only started playing recently): How is the app handling save games? Are those stored locally in the Browser Context and is there a way to retrieve those?

Did not find anything in the docs around that.

Appreciate taking the time to answer this!

**Comment by raz0red** [comment]
Hi-
Thanks a lot, I really appreciate it!

So, the handling is now done by persisting into the local browser database
(IndexedDB) for the site. You can view the save states by using the
browser's developer tools, looking in the "Applications" section, and then
under "Storage", you drill into "IndexedDB".

In there, you can see the saves. I don't believe there is an easy way to
export them. An upcoming feature is adding support for syncing saves to and
from the cloud (the user provides the target, Dropbox, etc.).

Thanks,
Chris

**Comment by PaulLuigi** [comment]
Is this local browser database ever cleared, for example if we clear our web history?

**Comment by raz0red** [comment, marked as answer]
Yeah, if you clear site data, they will be deleted. That is one reason the upcoming saves to Dropbox is being added.

**Comment by nn81** [comment]
Please consider support for something open source like NextCloud instead of a subscription-based service.

**Comment by raz0red** [comment]
Yeah, would like to support additional providers in the future. Is there a public NextCloud available or is it self-hosting?

Thanks,
Chris
